storage_service/templates/snippets/packages_table.html

```diff
@@ -9,11 +9,13 @@
         <th>Type</th>
         <th>Pointer File</th>
         <th>Status</th>
+        <th>Fixity Date</th>
+        <th>Fixity Status</th>
         <th>Actions</th>
       </tr>
     </thead>
     <tbody>
-    {% for package in packages %}
+    {% for last_fixity_check_datetime, last_fixity_check_result, package in package_data %}
```

Comment:
`last_fixity_check_datetime, last_fixity_check_result` are never provided as context anywhere - is this leftover from previous changes?